README.md

```diff
@@ -337,6 +354,13 @@ The following table lists the parameters for the `gui` component and their defau
 | `gui.service.loadBalancerSourceRanges` | Service load balancer source ranges | `nil` |
 | `gui.service.port` | Port for the service | `80` |
 | `gui.service.nodePort` | Node port for the service | `nil` |
+| `gui.httpPort` | Port for the HTTP listener in the container | `80` |
+| `gui.podSecurityContext.enabled` | Enable [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) | `false` |
+| `gui.podSecurityContext.runAsNonRoot` | Run as non-root user | `true` |
+| `gui.podSecurityContext.runAsUser` | User ID for the pod | `65534` |
+| `gui.containerSecurityContext.enabled` | Enable container [security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) | `false` |
+| `gui.containerSecurityContext.allowPrivilegeEscalation` | Allow privilege escalation for container | `false` |
+| `gui.containerSecurityContext.runAsUser` | User ID for the container | `65534` |
```

Comment:
Unfortunately the `gui` service is not supporting non-root users. Please set back to default for now.

Reply:
@oldgiova thank you for the detailed feedback. I would argue that we should give users the option to use a custom-built `gui` image which can run rootless like we do already. The change just adds options to configure the ports, but does not change the default. Also, the securityContext is disabled by default.

Reply:
> @oldgiova thank you for the detailed feedback. I would argue that we should give users the option to use a custom-built `gui` image which can run rootless like we do already. The change just adds options to configure the ports, but does not change the default. Also, the securityContext is disabled by default.

:thinking: yeah, you're right. Let's keep it then.